sdk/daml-lf/interpreter/BUILD.bazel

```diff
@@ -108,6 +110,23 @@ da_scala_test_suite(
     ],
 )
 
+da_scala_test(
+    name = "test-SBSECP256K1BoolTest",
```

Comment:
As the `SBSECP256K1Bool` builtin requires the BouncyCastle JCE for testing, we separate this specific test out